scenario_player/utils/token.py

```diff
@@ -399,19 +405,20 @@ def update_allowance(self) -> Union[Tuple[str, int], None]:
         required_allowance = self.config.settings.services.udc.token.balance_per_node * node_count
 
         log.debug(
```

Comment:
Should you use `local_log` here as well?

Reply:
Probably yeah, I'll make an issue about it.